modules/mono/editor/bindings_generator.h

```diff
@@ -614,6 +623,7 @@ class BindingsGenerator {
 	HashMap<const MethodInterface *, const InternalCall *> method_icalls_map;
 
 	HashMap<StringName, List<StringName>> blacklisted_methods;
+	HashSet<StringName> compat_singletons;
```

Comment:
while this isn't actually used by this PR yet, should this include a clear place to initialize this like `_initialize_blacklisted_methods`?

Reply:
I don't expect this to contain many elements so IMO just initializing it in `_initialize` is fine, I don't think the `_initialize_blacklisted_methods` method is needed either. But if you think it makes it clearer then yeah we can do it.

Reply:
I don't think they are necessary as dedicated methods either, but given that `_initialize_blacklisted_methods` already exists I think adding `_initialize_compat_singletons` makes sense.